vnext/ReactWindowsCore/WinRTWebSocketResource.h

```diff
@@ -20,9 +20,10 @@ namespace Microsoft::React {
 
 class WinRTWebSocketResource : public IWebSocketResource, public std::enable_shared_from_this<WinRTWebSocketResource> {
   winrt::Windows::Foundation::Uri m_uri;
-  winrt::Windows::Networking::Sockets::MessageWebSocket m_socket;
-  winrt::Windows::Networking::Sockets::MessageWebSocket::MessageReceived_revoker m_revoker;
-  winrt::Windows::Storage::Streams::DataWriter m_writer{m_socket.OutputStream()};
+  winrt::Windows::Networking::Sockets::IMessageWebSocket m_socket;
+  // TODO: Use or remove.
```

Comment:
>// TODO: Use or remove. [](start = 2, length = 23)

?? are you going to do this during this PR or future?

Reply:
Future. The member is not currently used, but it may if it yields a functional benefit.
I'll rule this out as I write read-related tests.